nuclio/triggers.py

```diff
@@ -225,24 +225,66 @@ class V3IOStreamTrigger(NuclioTrigger):
+                                       "Please use '{new_arg_name}' instead."
+
+        if "seekTo" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="seekTo",
+                                                            new_arg_name="seek_to"))
+            seek_to = deprecated_kwargs.get("seekTo")
+        if "pollingIntervalMS" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="pollingIntervalMS",
+                                                            new_arg_name="polling_interval_ms"))
+            polling_interval_ms = deprecated_kwargs.get("pollingIntervalMS")
+        if "readBatchSize" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="readBatchSize",
+                                                            new_arg_name="read_batch_size"))
+            read_batch_size = deprecated_kwargs.get("readBatchSize")
+        if "maxWorkers" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="max_workers",
+                                                            new_arg_name="max_workers"))
+            max_workers = deprecated_kwargs.get("maxWorkers")
+        if "sessionTimeout" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="sessionTimeout",
+                                                            new_arg_name="session_timeout"))
+            session_timeout = deprecated_kwargs.get("sessionTimeout")
+        if "workerAllocationMode" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="workerAllocationMode",
+                                                            new_arg_name="worker_allocation_mode"))
+            worker_allocation_mode = deprecated_kwargs.get("workerAllocationMode")
+        if "consumerGroup" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="consumerGroup",
+                                                            new_arg_name="consumer_group"))
+            consumer_group = deprecated_kwargs.get("consumerGroup")
+        if "sequenceNumCommitInterval" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="sequenceNumCommitInterval",
+                                                            new_arg_name="sequence_num_commit_interval"))
+            sequence_num_commit_interval = deprecated_kwargs.get("sequenceNumCommitInterval")
+        if "heartbeatInterval" in deprecated_kwargs:
+            logger.warn(deprecation_warning_template.format(old_arg_name="heartbeatInterval",
+                                                            new_arg_name="heartbeat_interval"))
+            heartbeat_interval = deprecated_kwargs.get("heartbeatInterval")
+
```

Comment:
- Move to a function that resolves the actual arguments
- Try to make it more general, maybe a for loop instead of an if tree

Reply:
As discussed, mark it as resolved since all options we discussed create another level of complexity.